### ISSUE TITLE
[For 10.0.9] Antivirus 0.13.0 release checklist

### ISSUE THREAD

**Issue by patrickjahns** [owncloud/files_antivirus#221]
### Reason
- Release needed for #219  and https://github.com/owncloud/files_antivirus/pull/222 and https://github.com/owncloud/files_antivirus/pull/226

### QA
- [x] Create release branch ``release/0.13.0`` and PR #238 @phil-davis 
- [x] Create "0.13.0 Testplan" #239  @phil-davis 
- [x] Test and document in testplan => Final "Go" from QA (set 'QA-ready' label) @phil-davis 

### Marketing
- [ ] Ping RocketChat channel #marketing
- [ ] Check/adapt info.xml stuff
- [ ] Check/adapt screenshot
- [ ] Blog post or other communication if applicable

### Documentation
- [ ] HTTP status for infected files has changed from 415 to 403 https://github.com/owncloud/documentation/issues/4271

### Build & Marketplace release

- [x] Bump version in info.xml- set `min-version 10.0.9` @patrickjahns
- [x] Update CHANGELOG.md - Change CHANGELOG.md 'Unreleased' to '$version' @patrickjahns
- [x] Create github tag for '$version' @patrickjahns
- [x] Build and sign tarball from github tag '$version' @patrickjahns
- [x] Upload to marketplace @patrickjahns
- [x] Send mail to release-coordination with link to this ticket and to the changelog @patrickjahns
- [x] Rename QA milestone to '$version' @patrickjahns
- [ ] Add a new 'Unreleased' section in CHANGELOG.md @patrickjahns
- [ ] Create PR to merge changes back to `master` @patrickjahns

### Specifics

- Open tickets before QA can start
- Other specifics for app release

**Comment by patrickjahns** [comment]
@VicDeo 
please add if there are other parts to be included in the next release

**Comment by VicDeo** [comment]
So far nothing but the fix for https://github.com/owncloud/core/issues/30751 that fixes https://github.com/owncloud/files_antivirus/issues/177 too

**Comment by VicDeo** [comment]
To document:
HTTP status for infected files has changed from 415 to 403

**Comment by VicDeo** [comment]
it would nice to have https://github.com/owncloud/files_antivirus/pull/226 too

**Comment by PVince81** [comment]
added https://github.com/owncloud/files_antivirus/pull/226 in OP

**Comment by patrickjahns** [comment]
what is the `min-version` for this app now related to the change of forbidden.

please also note - this needs to be at least a minor version - as we changed behavior - so instead of 0.12.1 - it is 0.13.0

**Comment by VicDeo** [comment]
@patrickjahns depends on 10.0.8 after #219

**Comment by davitol** [comment]
Tested partially in stable10 + files antivirus master branch using objectore and no new issues were found. https://github.com/owncloud/files_primary_s3/issues/61#issuecomment-394608063

**Comment by phil-davis** [comment]
Tested with core ``10.0.8``. With a bit of messing around copying backwards the core acceptance test code used by this app, I was able to get the acceptance tests to run.

With core ``10.0.8`` a ``500`` status gets returned when a virus is detected. The file upload is correctly blocked, so the functionality of blocking is fine.

With core ``10.0.9`` a ``403`` status gets returned when a virus is detected. (as per the acceptance test expectations) So that works more cleanly for a client that is caring to examine the status code of a failed upload.

**Comment by phil-davis** [comment]
@patrickjahns testing completed in #239 
Release PR #238 for review.
You should be able to just add a changelog to that PR and proceed with making the release...

**Comment by patrickjahns** [comment]
@phil-davis @PVince81 - raising `min-version` to `10.0.9` - `500` is unexpected behavior

**Comment by VicDeo** [comment]
@phil-davis not exactly.  it was 415 before.

**Comment by phil-davis** [comment]
OK - so I am glad that I made the effort to run tests against 10.0.8!
